src/components/VImageDetails/VRelatedImages.vue

```diff
@@ -50,6 +41,13 @@ export default {
     if (props.service) {
       relatedOptions.service = props.service
     }
+    /**
+     * Fetches related images on `imageId` change
+     * @param {object} props
+     * @param {string} props.imageId
+     * @param {any} props.service
+     * @return {{ images: Ref<ImageDetail[]> }}
+     */
```

Comment:
Just a heads up that this syntax will only work for a function definition, not a function call. Should we move this annotation to where `useRelated` is defined?